lib/internal/process/warning.js

```diff
@@ -78,6 +79,12 @@ function onWarning(warning) {
   if (typeof warning.detail === 'string') {
     msg += `\n${warning.detail}`;
   }
+  if (!trace && !traceWarningHelperShown) {
+    const flag = isDeprecation ? '--trace-deprecation' : '--trace-warnings';
+    msg += `\n(Use \`node ${flag} ...\` to show where the warning ` +
```

Comment:
Maybe we should either omit the binary name or at least extract it from `process.execPath` to avoid confusion if executing via a node binary not called `node`?

Reply:
@mscdex Done! Also opened # for doing this for --trace-uncaught.